addons/packages/multus-cni/3.7.1/README.md

```diff
@@ -1,24 +1,41 @@
 # Multus CNI
 
-This package provides the ability for enabling attaching multiple network interfaces to pods in Kubernetes using [multus-cni](https://github.com/k8snetworkplumbingwg/multus-cni).
+This package enables you to attach multiple network interfaces to pods in Kubernetes using [multus-cni](https://github.com/k8snetworkplumbingwg/multus-cni).
 
-## Supported Providers
+This documentation provides information about the specific TCE package. Please visit the [TCE package management page](https://tanzucommunityedition.io/docs/v0.11/package-management/) for general information about installation, removal, troubleshooting, and other topics.
 
-The following table shows the providers this package can work with.
+## Installation
 
-| AWS  |  Azure  | vSphere  | Docker |
-|:---:|:---:|:---:|:---:|
-| ✅  |  ✅  | ✅  | ✅ |
```

Comment:
I believe we wanted to standardize on having this supported provider table in each package's README. We may want to keep this part.

Reply:
Hi @stmcginnis , it was not deleted, but moved to a lower place.

I am following this doc that provided by @LappleApple to adjust the titles' sequence and its contents
https://docs.google.com/document/d/1WEZaKEBQ9A5xoOkbX4ZWREDvEw3zCwfu1oIrRWff8Y4/edit#heading=h.bzeiwt4lj7tb

I should add this doc to the pr description

Reply:
Chenrui is correct that the structure of the readme is changing.